Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -65,6 +66,49 @@ struct TestCommand: AsyncParsableCommand {
     )
     var retryCount: Int = 0
 
+    @Option(
+        name: .long,
+        help: "The test plan to run."
+    )
+    var testPlan: String?
+
+    @Option(
+        name: .long,
+        parsing: .upToNextOption,
+        help: "The list of test identifiers you want to test. Takes precedence over --skip-testing",
+        transform: TestIdentifier.init(string:)
+    )
+    var testTargets: [TestIdentifier] = []
+
+    @Option(
+        name: .long,
+        parsing: .upToNextOption,
+        help: "The list of test identifiers you want to skip testing.",
+        transform: TestIdentifier.init(string:)
+    )
+    var skipTestTargets: [TestIdentifier] = []
+
+    @Option(
+        name: .long,
+        parsing: .upToNextOption,
+        help: "The list of configurations you want to test. Takes precedence over --skip-test-configuration"
+    )
+    var testConfigurations: [String] = []
+
+    @Option(
+        name: .long,
+        parsing: .upToNextOption,
+        help: "The list of configurations you want to skip testing."
+    )
+    var skipTestConfigurations: [String] = []
```

Comment:
How does it work if you specify both of these?

Reply:
Not sure, I'll double check